### DISCUSSION TITLE
Добавить новый тип Input

### DISCUSSION THREAD

**Discussion by k-rustemuly** [moonshine-software/moonshine#463] [Ideas]
<img width="186" alt="KXD4C" src="https://github.com/moonshine-software/moonshine/assets/16911003/d6237c3b-0b6c-4b19-b62c-7a6fb18844e1">

**Comment by lee-to** [comment]
Добавим

**Comment by k-rustemuly** [comment]
![изображение](https://github.com/moonshine-software/moonshine/assets/16911003/b5849ba7-c34d-4e52-9a92-a2ac3ddf664f)
И такие еще можно? типа поменять label switch

**Reply by alexvenga** [reply]
Так оно есть
Switch Boolean

**Comment by Opekunov** [comment]
> <img alt="KXD4C" width="186" src="https://user-images.githubusercontent.com/16911003/265550318-d6237c3b-0b6c-4b19-b62c-7a6fb18844e1.png">

https://github.com/moonshine-software/moonshine/releases/tag/2.0.0-rc.2
